src/main/java/org/carlmontrobotics/subsystems/IntakeShooter.java

```diff
@@ -202,8 +202,8 @@ public void setMaxIntake(int direction) {
 
     }
 
-    public void setMaxOuttake(int direction) {
-        outtakeMotorVortex.set(1 * direction);
+    public void setMaxOuttake(double d) {
+        outtakeMotorVortex.set(1 * d);
```

Comment:
`setMaxOuttake` is an inappropriate name since you don't necessarily go at "max" speed. 
Also, it's generally bad practice to change the functionality of an existing method b/c you can end up breaking the code that uses the original method (Luckily your change didn't do that). Unless you have a good reason to edit the original method, create a new method instead.